### ISSUE TITLE
Other Webhooks

### ISSUE THREAD

**Issue by MrKeiKun** [nTraum/gitlab-irc#6]
Any plans of implementing other webhooks
like 
- Issues
- Merge Request
- Tagging
  ??

**Comment by nTraum** [comment]
Yeah, I'd like to support the other webhooks as well. No ETA at the moment, will do when I find time for it. Pull requests are welcomed of course. ;)

**Comment by ochorocho** [comment]
Hey ... my fork does support some more webhooks.
It takes care of the following:
- Issues new/updated
- Notes on issues/code

https://github.com/ochorocho/gitlab-irc

**Comment by MrKeiKun** [comment]
@ochorocho 
Yours forces me to run it over root.
